### PR TITLE
Update configuration-with-ocis.md

### PR DESCRIPTION
**Pull request by LukasHirt**
Since we're registering the greet endpoint in the proxy, we should also use that in the config for the extension in the UI, right? Feel free to close this if it's a nonsense

### CODE REVIEW

**Comment by refs** [comment]
I think we actually removed the ocis-hello references from the proxy. See https://github.com/owncloud/ocis/commit/771f85ce2bfbe55a25d3c858a2c5c9f861eff719

File: docs/configuration-with-ocis.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ After that we will need to create a config file for phoenix so that we can load
             "id": "hello",
             "path": "http://localhost:9105/hello.js",
```
Comment (kulmann) [inline docs/configuration-with-ocis.md:56]: Please also change the path here to 
```suggestion
            "path": "https://localhost:9200/hello.js",
```

**Review by kulmann — CHANGES_REQUESTED** [review]
You're right, since ocis-hello depends on having the settings service available it makes sense that the default routing points to the proxy.
Could you also add to the documentation, that the proxy config also needs the following entry:
```
{
          "endpoint": "/hello.js",
          "backend": "http://localhost:9105"
        }
```

**Comment by LukasHirt** [comment]
@kulmann Done

**Comment by LukasHirt** [comment]
> I think we actually removed the ocis-hello references from the proxy

We still mention in the docs of ocis-hello to add it manually to the config of proxy

**Review by kulmann — APPROVED** [review]
Thanks for updating this! :heart:

**Review by refs — APPROVED** [review]

**Comment by wkloucek** [comment]
@LukasHirt  should be already reflected in #https://github.com/owncloud/ocis-hello/pull/104

**Comment by LukasHirt** [comment]
> should be already reflected in #104 ?

Ah, right, thank you! I like your approach with omitting the address completely much more :rocket: Closing this in favour of that PR

**Pull request closed without merging**